diff -Naur wget-1.21.3/lib/cdefs.h wget-1.21.3.tpg/lib/cdefs.h
--- wget-1.21.3/lib/cdefs.h	2022-02-26 16:22:12.000000000 +0000
+++ wget-1.21.3.tpg/lib/cdefs.h	2022-05-26 21:15:12.285799294 +0000
@@ -685,7 +685,7 @@
 #  define __attr_access_none(argno)
 #endif
 
-#if __GNUC_PREREQ (11, 0)
+#if __GNUC_PREREQ (11, 0) && !defined(__clang__)
 /* Designates dealloc as a function to call to deallocate objects
    allocated by the declared function.  */
 # define __attr_dealloc(dealloc, argno) \
diff -Naur wget-1.21.3/lib/dirent.in.h wget-1.21.3.tpg/lib/dirent.in.h
--- wget-1.21.3/lib/dirent.in.h	2022-02-26 16:22:12.000000000 +0000
+++ wget-1.21.3.tpg/lib/dirent.in.h	2022-05-26 21:15:47.075638896 +0000
@@ -59,7 +59,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.3/lib/stdio.in.h wget-1.21.3.tpg/lib/stdio.in.h
--- wget-1.21.3/lib/stdio.in.h	2022-02-26 16:22:13.000000000 +0000
+++ wget-1.21.3.tpg/lib/stdio.in.h	2022-05-26 21:16:31.395434534 +0000
@@ -95,7 +95,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.3/lib/stdlib.in.h wget-1.21.3.tpg/lib/stdlib.in.h
--- wget-1.21.3/lib/stdlib.in.h	2022-02-26 16:22:13.000000000 +0000
+++ wget-1.21.3.tpg/lib/stdlib.in.h	2022-05-26 21:16:51.385342351 +0000
@@ -103,7 +103,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.3/m4/gnulib-common.m4 wget-1.21.3.tpg/m4/gnulib-common.m4
--- wget-1.21.3/m4/gnulib-common.m4	2022-02-26 16:22:14.000000000 +0000
+++ wget-1.21.3.tpg/m4/gnulib-common.m4	2022-05-26 21:17:53.805057097 +0000
@@ -178,7 +178,7 @@
    _GL_ATTRIBUTE_DEALLOC_FREE declares that the function returns pointers that
    can be freed via 'free'; it can be used only after declaring 'free'.  */
 /* Applies to: functions.  Cannot be used on inline functions.  */
-#if _GL_GNUC_PREREQ (11, 0)
+#if _GL_GNUC_PREREQ (11, 0) && !defined(__clang__)
 # define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 #else
 # define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.3/src/config.h.in wget-1.21.3.tpg/src/config.h.in
--- wget-1.21.3/src/config.h.in	2022-02-26 16:23:44.000000000 +0000
+++ wget-1.21.3.tpg/src/config.h.in	2022-05-26 21:18:22.394926958 +0000
@@ -2635,7 +2635,7 @@
    _GL_ATTRIBUTE_DEALLOC_FREE declares that the function returns pointers that
    can be freed via 'free'; it can be used only after declaring 'free'.  */
 /* Applies to: functions.  Cannot be used on inline functions.  */
-#if _GL_GNUC_PREREQ (11, 0)
+#if _GL_GNUC_PREREQ (11, 0) && !defined(__clang__)
 # define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 #else
 # define _GL_ATTRIBUTE_DEALLOC(f, i)
