diff -up wget-1.20.1/src/init.c.omv~ wget-1.20.1/src/init.c
--- wget-1.20.1/src/init.c.omv~	2019-01-10 16:50:39.272074578 +0100
+++ wget-1.20.1/src/init.c	2019-01-10 16:55:34.983038319 +0100
@@ -408,6 +408,7 @@ defaults (void)
   opt.ftp_glob = true;
   opt.htmlify = true;
   opt.http_keep_alive = true;
+  opt.content_disposition = true;
   opt.use_proxy = true;
   opt.convert_file_only = false;
   tmp = getenv ("no_proxy");
diff -up wget-1.20.1/doc/wget.info.omv~ wget-1.20.1/doc/wget.info
--- wget-1.20.1/doc/wget.info.omv~	2019-02-11 22:17:20.676340374 +0100
+++ wget-1.20.1/doc/wget.info	2019-02-11 22:17:57.853907003 +0100
@@ -1479,8 +1479,8 @@ File: wget.info,  Node: HTTP Options,  N
      If this is set to on, experimental (not fully-functional) support
      for ‘Content-Disposition’ headers is enabled.  This can currently
      result in extra round-trips to the server for a ‘HEAD’ request, and
-     is known to suffer from a few bugs, which is why it is not
-     currently enabled by default.
+     is known to suffer from a few bugs. It is enabled by default.
+     Use ‘--no-content-disposition‘ to turn it off.
 
      This option is useful for some file-downloading CGI programs that
      use ‘Content-Disposition’ headers to describe what the name of a
diff -up wget-1.20.1/doc/wget.texi.omv~ wget-1.20.1/doc/wget.texi
--- wget-1.20.1/doc/wget.texi.omv~	2019-02-11 22:16:31.864596416 +0100
+++ wget-1.20.1/doc/wget.texi	2019-02-11 22:17:18.270303704 +0100
@@ -1721,7 +1721,8 @@ specified under @samp{--post-data} are f
 If this is set to on, experimental (not fully-functional) support for
 @code{Content-Disposition} headers is enabled. This can currently result in
 extra round-trips to the server for a @code{HEAD} request, and is known
-to suffer from a few bugs, which is why it is not currently enabled by default.
+to suffer from a few bugs. It is enabled by default. Use
+@samp{--no-content-disposition} to turn it off.
 
 This option is useful for some file-downloading CGI programs that use
 @code{Content-Disposition} headers to describe what the name of a
