diff -Naur wget-1.21.4/lib/cdefs.h wget-1.21.4.tpg/lib/cdefs.h
--- wget-1.21.4/lib/cdefs.h	2023-05-11 00:53:31.000000000 +0000
+++ wget-1.21.4.tpg/lib/cdefs.h	2023-06-01 17:11:17.022014311 +0000
@@ -693,7 +693,7 @@
 #  define __attr_access_none(argno)
 #endif
 
-#if __GNUC_PREREQ (11, 0)
+#if __GNUC_PREREQ (11, 0) && !defined(__clang__)
 /* Designates dealloc as a function to call to deallocate objects
    allocated by the declared function.  */
 # define __attr_dealloc(dealloc, argno) \
diff -Naur wget-1.21.4/lib/dirent.in.h wget-1.21.4.tpg/lib/dirent.in.h
--- wget-1.21.4/lib/dirent.in.h	2023-05-11 00:53:31.000000000 +0000
+++ wget-1.21.4.tpg/lib/dirent.in.h	2023-06-01 17:12:11.195134653 +0000
@@ -78,7 +78,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.4/lib/stdio.in.h wget-1.21.4.tpg/lib/stdio.in.h
--- wget-1.21.4/lib/stdio.in.h	2023-05-11 00:53:32.000000000 +0000
+++ wget-1.21.4.tpg/lib/stdio.in.h	2023-06-01 17:12:44.335004628 +0000
@@ -112,7 +112,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.4/lib/stdlib.in.h wget-1.21.4.tpg/lib/stdlib.in.h
--- wget-1.21.4/lib/stdlib.in.h	2023-05-11 00:53:32.000000000 +0000
+++ wget-1.21.4.tpg/lib/stdlib.in.h	2023-06-01 17:13:00.644940705 +0000
@@ -107,7 +107,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.4/m4/gnulib-common.m4 wget-1.21.4.tpg/m4/gnulib-common.m4
--- wget-1.21.4/m4/gnulib-common.m4	2023-05-11 00:53:33.000000000 +0000
+++ wget-1.21.4.tpg/m4/gnulib-common.m4	2023-06-01 17:13:36.914798713 +0000
@@ -212,7 +212,7 @@
    can be freed via 'free'; it can be used only after declaring 'free'.  */
 /* Applies to: functions.  Cannot be used on inline functions.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if _GL_GNUC_PREREQ (11, 0)
+# if _GL_GNUC_PREREQ (11, 0) && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -Naur wget-1.21.4/src/config.h.in wget-1.21.4.tpg/src/config.h.in
--- wget-1.21.4/src/config.h.in	2023-05-11 00:55:01.000000000 +0000
+++ wget-1.21.4.tpg/src/config.h.in	2023-06-01 17:13:58.301381751 +0000
@@ -2716,7 +2716,7 @@
    can be freed via 'free'; it can be used only after declaring 'free'.  */
 /* Applies to: functions.  Cannot be used on inline functions.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if _GL_GNUC_PREREQ (11, 0)
+# if _GL_GNUC_PREREQ (11, 0) && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
