diff -urN wget-1.18/src/ftp.c wget-1.18-patched/src/ftp.c
--- wget-1.18/src/ftp.c	2016-06-10 02:10:14.000000000 +1000
+++ wget-1.18-patched/src/ftp.c	2016-07-10 15:04:38.840204226 +1000
@@ -1427,7 +1427,7 @@
           fp = fopen (con->target, "ab");
 #endif /* def __VMS [else] */
         }
-      else if (opt.noclobber || opt.always_rest || opt.timestamping || opt.dirstruct
+      else if (opt.forceclobber || opt.noclobber || opt.always_rest || opt.timestamping || opt.dirstruct
                || opt.output_document || count > 0)
         {
           if (opt.unlink_requested && file_exists_p (con->target, NULL))
diff -urN wget-1.18/src/init.c wget-1.18-patched/src/init.c
--- wget-1.18/src/init.c	2016-06-03 21:55:13.000000000 +1000
+++ wget-1.18-patched/src/init.c	2016-07-10 15:04:38.841204226 +1000
@@ -192,6 +192,7 @@
   { "excludedomains",   &opt.exclude_domains,   cmd_vector },
   { "followftp",        &opt.follow_ftp,        cmd_boolean },
   { "followtags",       &opt.follow_tags,       cmd_vector },
+  { "forceclobber",     &opt.forceclobber,      cmd_boolean },
   { "forcehtml",        &opt.force_html,        cmd_boolean },
   { "ftppasswd",        &opt.ftp_passwd,        cmd_string }, /* deprecated */
   { "ftppassword",      &opt.ftp_passwd,        cmd_string },
diff -urN wget-1.18/src/main.c wget-1.18-patched/src/main.c
--- wget-1.18/src/main.c	2016-06-07 01:36:04.000000000 +1000
+++ wget-1.18-patched/src/main.c	2016-07-10 15:04:38.843204226 +1000
@@ -302,6 +302,7 @@
     { "execute", 'e', OPT__EXECUTE, NULL, required_argument },
     { "follow-ftp", 0, OPT_BOOLEAN, "followftp", -1 },
     { "follow-tags", 0, OPT_VALUE, "followtags", -1 },
+    { "force-clobber", 0, OPT_BOOLEAN, "forceclobber", -1 },
     { "force-directories", 'x', OPT_BOOLEAN, "dirstruct", -1 },
     { "force-html", 'F', OPT_BOOLEAN, "forcehtml", -1 },
     { "ftp-password", 0, OPT_VALUE, "ftppassword", -1 },
@@ -618,6 +619,8 @@
     N_("\
   -O,  --output-document=FILE      write documents to FILE\n"),
     N_("\
+  --force-clobber                  clobber existing files.\n"),
+    N_("\
   -nc, --no-clobber                skip downloads that would download to\n\
                                      existing files (overwriting them)\n"),
     N_("\
@@ -1463,7 +1466,21 @@
   if (opt.timestamping && opt.noclobber)
     {
       fprintf (stderr, _("\
-Can't timestamp and not clobber old files at the same time.\n"));
+                         Can't timestamp and not clobber old files at the same time.\n"));
+      print_usage (1);
+      exit (1);
+    }
+    if (opt.timestamping && opt.forceclobber)
+    {
+      printf (_("\
+                +Can't timestamp and force clobber old files at the same time.\n"));
+      print_usage (1);
+      exit (1);
+    }
+    if (opt.forceclobber && opt.noclobber)
+    {
+      printf (_("\
+                +Can't force clobber and not clobber old files at the same time.\n"));
       print_usage (1);
       exit (WGET_EXIT_GENERIC_ERROR);
     }
diff -urN wget-1.18/src/options.h wget-1.18-patched/src/options.h
--- wget-1.18/src/options.h	2016-06-03 21:55:13.000000000 +1000
+++ wget-1.18-patched/src/options.h	2016-07-10 15:04:38.843204226 +1000
@@ -60,6 +60,7 @@
   int cut_dirs;                 /* Number of directory components to cut. */
   bool add_hostdir;             /* Do we add hostname directory? */
   bool protocol_directories;    /* Whether to prepend "http"/"ftp" to dirs. */
+  int forceclobber;             /* Forces clobbering of existing files. */
   bool noclobber;               /* Disables clobbering of existing data. */
   bool unlink_requested;        /* remove file before clobbering */
   char *dir_prefix;             /* The top of directory tree */
diff -urN wget-1.18/src/url.c wget-1.18-patched/src/url.c
--- wget-1.18/src/url.c	2016-03-03 18:39:56.000000000 +1000
+++ wget-1.18-patched/src/url.c	2016-07-10 15:04:38.844204226 +1000
@@ -1790,6 +1790,7 @@
   xfree (temp_fnres.base);
 
   /* Check the cases in which the unique extensions are not used:
+     0) Clobbering is forced (--force-clobber)
      1) Clobbering is turned off (-nc).
      2) Retrieval with regetting.
      3) Timestamping is used.
diff -urN wget-1.18/src/url.h wget-1.18-patched/src/url.h
--- wget-1.18/src/url.h	2016-01-19 19:26:57.000000000 +1000
+++ wget-1.18-patched/src/url.h	2016-07-10 15:04:38.844204226 +1000
@@ -51,7 +51,7 @@
 /* The flags that allow clobbering the file (opening with "wb").
    Defined here to avoid repetition later.  #### This will require
    rework.  */
-#define ALLOW_CLOBBER (opt.noclobber || opt.always_rest || opt.timestamping \
+#define ALLOW_CLOBBER (opt.forceclobber || opt.noclobber || opt.always_rest || opt.timestamping \
                   || opt.dirstruct || opt.output_document || opt.backups > 0)
 
 /* Specifies how, or whether, user auth information should be included
