diff --git a/src/ftp.c b/src/ftp.c
index a3b2cb6..54f357c 100644
--- a/src/ftp.c
+++ b/src/ftp.c
@@ -1234,7 +1234,7 @@ Error in server response, closing control connection.\n"));
           fp = fopen (con->target, "ab");
 #endif /* def __VMS [else] */
         }
-      else if (opt.noclobber || opt.always_rest || opt.timestamping || opt.dirstruct
+      else if (opt.forceclobber || opt.noclobber || opt.always_rest || opt.timestamping || opt.dirstruct
                || opt.output_document || count > 0)
         {
           if (opt.unlink && file_exists_p (con->target))
diff --git a/src/init.c b/src/init.c
index 088a6e9..4b74e7a 100644
--- a/src/init.c
+++ b/src/init.c
@@ -183,6 +183,7 @@ static const struct {
   { "excludedomains",   &opt.exclude_domains,   cmd_vector },
   { "followftp",        &opt.follow_ftp,        cmd_boolean },
   { "followtags",       &opt.follow_tags,       cmd_vector },
+  { "forceclobber",     &opt.forceclobber,      cmd_boolean },
   { "forcehtml",        &opt.force_html,        cmd_boolean },
   { "ftppasswd",        &opt.ftp_passwd,        cmd_string }, /* deprecated */
   { "ftppassword",      &opt.ftp_passwd,        cmd_string },
diff --git a/src/main.c b/src/main.c
index 99c2819..123db56 100644
--- a/src/main.c
+++ b/src/main.c
@@ -211,6 +211,7 @@ static struct cmdline_option option_data[] =
     { "execute", 'e', OPT__EXECUTE, NULL, required_argument },
     { "follow-ftp", 0, OPT_BOOLEAN, "followftp", -1 },
     { "follow-tags", 0, OPT_VALUE, "followtags", -1 },
+    { "force-clobber", 0, OPT_BOOLEAN, "forceclobber", -1 },
     { "force-directories", 'x', OPT_BOOLEAN, "dirstruct", -1 },
     { "force-html", 'F', OPT_BOOLEAN, "forcehtml", -1 },
     { "ftp-password", 0, OPT_VALUE, "ftppassword", -1 },
@@ -500,6 +501,8 @@ Download:\n"),
     N_("\
   -O,  --output-document=FILE      write documents to FILE.\n"),
     N_("\
+  --force-clobber                clobber existing files.\n"),
+    N_("\
   -nc, --no-clobber                skip downloads that would download to\n\
                                    existing files (overwriting them).\n"),
     N_("\
@@ -1286,6 +1289,20 @@ main (int argc, char **argv)
       fprintf (stderr, _("\
 Can't timestamp and not clobber old files at the same time.\n"));
       print_usage (1);
+      exit (1);
+    }
+    if (opt.timestamping && opt.forceclobber)
+    {
+      printf (_("\
+                +Can't timestamp and force clobber old files at the same time.\n"));
+      print_usage (1);
+      exit (1);
+    }
+    if (opt.forceclobber && opt.noclobber)
+    {
+      printf (_("\
+                +Can't force clobber and not clobber old files at the same time.\n"));
+      print_usage (1);
       exit (WGET_EXIT_GENERIC_ERROR);
     }
 #ifdef ENABLE_IPV6
diff --git a/src/options.h b/src/options.h
index b995126..344426f 100644
--- a/src/options.h
+++ b/src/options.h
@@ -53,6 +53,7 @@ struct options
   int cut_dirs;                 /* Number of directory components to cut. */
   bool add_hostdir;             /* Do we add hostname directory? */
   bool protocol_directories;    /* Whether to prepend "http"/"ftp" to dirs. */
+  int forceclobber;            /* Forces clobbering of existing files. */
   bool noclobber;               /* Disables clobbering of existing data. */
   bool unlink;                  /* remove file before clobbering */
   char *dir_prefix;             /* The top of directory tree */
diff --git a/src/url.c b/src/url.c
index 50ab0ed..9c1e0d9 100644
--- a/src/url.c
+++ b/src/url.c
@@ -1667,6 +1667,7 @@ url_file_name (const struct url *u, char *replaced_filename)
   xfree (temp_fnres.base);
 
   /* Check the cases in which the unique extensions are not used:
+     0) Clobbering is forced (--force-clobber)
      1) Clobbering is turned off (-nc).
      2) Retrieval with regetting.
      3) Timestamping is used.
diff --git a/src/url.h b/src/url.h
index 1627be9..1e25193 100644
--- a/src/url.h
+++ b/src/url.h
@@ -50,7 +50,7 @@ as that of the covered work.  */
 /* The flags that allow clobbering the file (opening with "wb").
    Defined here to avoid repetition later.  #### This will require
    rework.  */
-#define ALLOW_CLOBBER (opt.noclobber || opt.always_rest || opt.timestamping \
+#define ALLOW_CLOBBER (opt.forceclobber || opt.noclobber || opt.always_rest || opt.timestamping \
                   || opt.dirstruct || opt.output_document || opt.backups > 0)
 
 /* Specifies how, or whether, user auth information should be included
